dev-tools/packer/platforms/centos/beatname.sh.j2

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Script to run {.beat_name} in foreground with the same path settings that
+# the init script / systemd unit file would do.
+
+/usr/share/{{.beat_name}}/bin/{{.beat_name}} -e \
+  -c /etc/{{.beat_name}}/{{.beat_name}}.yml \
+  -path.home /usr/share/{{.beat_name}} \
+  -path.config /etc/{{.beat_name}} \
+  -path.data /var/lib/{{.beat_name}} \
+  $@
```

Comment:
Should this be `"$@"` instead of `$@`? It affects how certain arguments get passed. [StackOverflow](https://stackoverflow.com/questions/4824590/propagate-all-arguments-in-a-bash-shell-script)

```
macbook13:tmp akroh$ cat foo.sh 
#!/bin/bash
./baz.sh $@
macbook13:tmp akroh$ cat bar.sh 
#!/bin/bash
./baz.sh "$@"
macbook13:tmp akroh$ cat baz.sh 
#!/bin/bash
echo Received: $1
echo Received: $2
echo Received: $3
echo Received: $4

macbook13:tmp akroh$ ./foo.sh -c="andrews config.yml"
Received: -c=andrews
Received: config.yml
Received:
Received:
macbook13:tmp akroh$ ./bar.sh -c="andrews config.yml"
Received: -c=andrews config.yml
Received:
Received:
Received:
```